internal/updater/updater.go

```diff
@@ -88,7 +88,7 @@ func (u *AvailableUpdate) Equals(origin *Origin) bool {
 	return u.Channel == origin.Channel && u.Version == origin.Version
 }
 
-type Update struct {
+type UpdateInstall struct {
```

Comment:
```suggestion
type UpdateInstaller struct {
```

`UpdateInstall` suggests to me that it holds information about an update, which is exactly the confusion this rename was meant to address.